### ISSUE TITLE
Authorization Failed for Deployed Chatbot

### ISSUE THREAD

**Issue by jessegoraya** [Microsoft/BotBuilder#2632]
## System Information
Language: C#
SDK: 3.5.5
Development environment - Localhost and Azure App Service

## Issue Description
 I am having difficulty with a chatbot that I developed which works fine locally but after it was deployed to dev.botframework.com it does not appear to work.  I get the error{"Authorization for Microsoft App ID a8641a16-932c-49a5-af8b-a58ab2ce251f failed with status code Unauthorized and reason phrase 'Unauthorized'"}

## Example Code

See [Stackoverflow for code](http://stackoverflow.com/questions/43540575/authorization-failed-for-deployed-chatbot) 

## Steps to Reproduce
I have tried the instructions at[ Troubleshooting Bot Framework Authentication ](https://docs.botframework.com/en-us/support/troubleshooting-bot-framework-authentication/) with the following results:

Step 1: Connect without password on localhost - Worked fine!
Step 2: Verify AppID and Password are Correct - They are!
Step 3: Enable security and run on localhost - This does not work :( However the endpoint is correct as are the MicrosoftAppID and Password
Step 4: Connect to your bot using the Bot Framework Developer Portal - This also works! However when using the web chat feature within the portal I also get an internal server error
Any ideas on how best to resolve would really be helpful!

## Actual Results

My code is in the stackoverflow link below but it breaks at the line... await Conversation.SendAsync(activity, () => new MyBot.AppServices.ServiceLUIS()); where it states {"Authorization for Microsoft App ID a8641a16-932c-49a5-af8b-a58ab2ce251f failed with status code Unauthorized and reason phrase 'Unauthorized'"}

StackOverflow Post with Code
[See StackOverflow](http://stackoverflow.com/questions/43540575/authorization-failed-for-deployed-chatbot)

Bottom Line
So it runs locally on the emulator without the password, the end point check works with the Dev Botframework Portal but both the Web Chat and the Emulator return internal server errors.  Where it is breaking is on a call to my LUIS service so to me it seems like my Bot doesn't have authorization to call my LUIS model.  However I can't seem to find anything in Azure where my bot is published and LUIS Model resides where I would allow LUIS model to authorize access to my BOT.  Also don't see anything in Luis.ai or Bot Framework Portal which would provide authorization from the Bot to the model or vice versa.

**Comment by jessegoraya** [comment]
Solved! Ended up creating another new key and password for the bot and republishing and it worked! –
